backend/app/controllers/api/Blobs.scala

```diff
@@ -23,21 +23,25 @@ class Blobs(override val controllerComponents: AuthControllerComponents, manifes
   def getBlobs = ApiAction.attempt { req =>
     checkPermission(CanPerformAdminOperations, req) {
       (param("collection", req), param("ingestion", req)) match {
-        case (Some(collection), Some(ingestion)) =>
-          val uri = Uri(collection).chain(ingestion)
+        case (Some(collection), maybeIngestion) =>
+          val parentExists = maybeIngestion match {
```

Comment:
could you add an explicit type here? getINgestion/getCollection sound like they might return an object but parentExists looks like a boolean

Reply:
Yeah, fair point. I couldn't really understand why Michael did `manifest.getIngestion(uri)` and then threw away the result. My best interpretation was that he's checking whether the ingestion exists before counting its blobs. (Whether this is actually necessary, I haven't checked).

So I tried to rename to `parentExists` to make it clearer, but you're right, it sounds like it contains a boolean, when actually it's just being use as a boolean

Reply:
I think I might just delete that whole initial check for whether the collection/ingestion exists - don't see why it's necessary since the ES query should just return empty in such a case